### PR TITLE
Added docker-ce and meta-ci.os configurations and Azure iotedge

### PR DESCRIPTION
**Pull request by siredmar**
Added docker modifications

This PR accompanies https://github.com/ci4rail/meta-ci.os/pull/5

Merge this after https://github.com/ci4rail/meta-ci.os/pull/5 has been merged.

This PR fixes https://ci4rail.atlassian.net/browse/KYT-193 and https://ci4rail.atlassian.net/browse/KYT-190

### CODE REVIEW

File: mender-on-verdin/kasfile.yaml [hunk 1]
```diff
@@ -94,6 +94,11 @@ repos:
     refspec: 2120d69c3d63405da5dcba2cefc678944b85a582
     layers:
       meta-mender-toradex-nxp:
+  
+  # docker
+  src/meta-virtualization:
+    url: git://git.yoctoproject.org/meta-virtualization
```
Comment (KlausPopp) [inline mender-on-verdin/kasfile.yaml:100]: Add git commit id here

**Review by KlausPopp — CHANGES_REQUESTED** [review]
We should switch meta-mender to release dunfell-v2021.01

File: mender-on-verdin/kasfile.yaml (hunk 1 above)
Comment (siredmar) [reply mender-on-verdin/kasfile.yaml:100]: version now set to `dunfell-v2021.01`

Comment (KlausPopp) [reply mender-on-verdin/kasfile.yaml:100]: I meant: Use a commit id for meta-virtualization (not just the branch name)

**Review by KlausPopp — COMMENTED** [review]
Update README.md: 
Document iotedge setup: device_connection_string must be manually configured in /etc/iotedge/config.yaml. (to get device connection string, see https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux)

File: mender-on-verdin/kasfile.yaml (hunk 1 above)
Comment (siredmar) [reply mender-on-verdin/kasfile.yaml:100]: oh, i misunderstood. Ok.

**Comment by siredmar** [comment]
> Update README.md:
> Document iotedge setup: device_connection_string must be manually configured in /etc/iotedge/config.yaml. (to get device connection string, see https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux)

Readme modified.

**Review by KlausPopp — APPROVED** [review]

**Pull request merged**